cmd/kubeadm/app/util/marshal.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	clientsetscheme "k8s.io/client-go/kubernetes/scheme"
+)
+
+// Marshal object into yaml.
+func MarshalToYaml(obj runtime.Object, gv schema.GroupVersion) ([]byte, error) {
```

Comment:
we're enforcing golint now; you must have a valid comment

Reply:
cc @sttts Should/can we move this somewhere else?

Reply:
What's special requirement here? Why is the comments considered not valid?

Reply:
@medinatiger it needs to follow godoc conventions, where the first word matches the function name (`MarshalToYaml marshals an object into yaml.`).

Reply:
@ncdc, thanks! Updated.